apps/web/src/utils/slugs.tsx

```diff
@@ -0,0 +1,8 @@
+export function slugify(text: string) {
```

Comment:
Consider using slugify npm package because it handles locales too

Reply:
Godt poeng, fikser